frontend/src/Onboarding/OnboardingInfoScreen.tsx

```diff
@@ -180,11 +200,14 @@ class OnboardingScreenComponent extends React.Component<
    * Renders the major drop down
    */
   renderMajorDropDown() {
+    let majorNames = this.props.majors.filter(
+      major => major.yearVersion === this.state.catalogYear
```

Comment:
Do all of our majors have a yearVersion? It might be safer to show majors with no year version and those with the same year version.

Also, is there a major for every year version? Is there the case where a major has a year version, let's say 2018, and there does not exist a 2019 year version for that same major. The user selects a catalog year of 2019. What happens then? 

I'm not too familiar with how the majors work and such but just want to make sure that we handle all of these cases (if they are possible)

Reply:
Yup! All majors will have a year version, and majors can not exist on certain years (ex information science no longer exists and several data science combined majors have been added). I'll let Nathan or Jmoney respond about the code behavior but just for the context

Reply:
If there is no year version for that major, it will not show up in the drop down. To go off of Iman's example, if you pick 2020 as the catalog year, you won't see Information Science since it doesn't exist anymore and will be filtered out. I think it would make sense for this to be the expected behavior.